maxrefdes178-FaceId/maxrefdes178_max32666/src/max32666_data.c

```diff
@@ -69,7 +69,7 @@ volatile device_settings_t device_settings = {
         .enable_max78000_audio = 1,
         .enable_max78000_audio_cnn = 1,
         .enable_max78000_video = 0,
-        .enable_max78000_video_cnn = 0,
+        .enable_max78000_video_cnn = 1,
```

Comment:
enable_max78000_video_cnn default value should match with enable_cnn default value in max78000_video_main.c

Reply:
updated